addons/website/views/snippets/snippets.xml

```diff
@@ -294,6 +294,23 @@
         t-att-data-exclude="bg_exclude">
         <t t-call="web_editor.image_optimization_controls"/>
     </div>
+    <div data-js="BackgroundShape" string="Shape"
+        data-selector="section, .parallax, .carousel-item"
+        data-exclude=".s_hr, .s_image_gallery, .o_gallery .carousel-item">
```

Comment:
Why not using the pretty variable above?

Reply:
it didn't exist at the time, fixed